src/main/resources/postrm.ftl

```diff
@@ -4,11 +4,9 @@ if [ "$1" = "purge" ] ; then
 fi
 </#if>
 
-USER_HOME=`echo ~${config.user}`
-
 case "$1" in
         purge|remove)
-                rm -R $USER_HOME/${config.artifactId}
+                rm -R /${config.installDir}
```

Comment:
I'm seeing you put quotes (") around config.installDir, which makes perfect sense. It makes sense put them everywhere. Just for consistency.

Reply:
Done